test/Lava.spec.ts

```diff
@@ -0,0 +1,81 @@
+import 'phaser';
+import PhaserTester from './lib/PhaserTester';
+import { LavaScene } from './scenes/LavaScene';
+import { assert, expect } from 'chai';
+import { Lava } from '../src/gameobjects/Lava';
+import { Player } from '../src/gameobjects/Player';
+
+describe('Lava', () => {
+    let scene: LavaScene;
+    let lava: Lava;
+    let playerMock: Player;
+    let phaserTester: PhaserTester<LavaScene>;
+
+    before(async () => {
+        phaserTester = new PhaserTester(LavaScene);
+        await phaserTester.setup();
+        scene = phaserTester.getScene();
+    });
+
+    beforeEach(function () {
+        scene.onTestStart((_lava, _player) => {
+            lava = _lava;
+            playerMock = _player;
+        });
+        phaserTester.onTestStart(this.currentTest.title);
+    });
+
+    it('should move up', async () => {
+        const oldLavaY: number = lava.y;
+        await phaserTester.delay(1000);
+        expect(lava.y).to.be.lessThan(oldLavaY);
+    });
+
+    it('should be able to speed up after 10 seconds', async () => {
+        let oldLavaY: number = lava.y;
```

Comment:
Can be flaky
```
1) should be able to speed up after 10 seconds
 
     Lava
 
15 06 2020 05:49:13.577:WARN [reporter]: SourceMap position not found for trace: AssertionError: expected 45.08331750021165 to be above 45.670042499482946
 
    at Object.<anonymous> (base/test/Lava.spec.js?59f07e7a040ef901a70b8b4962f0802e56e327b5:44:58)
 
    at Generator.next (<anonymous>)
 
    at fulfilled (base/test/Lava.spec.js?59f07e7a040ef901a70b8b4962f0802e56e327b5:4:58)
 
     AssertionError: expected 45.08331750021165 to be above 45.670042499482946
 
    at Object.<anonymous> (test/Lava.spec.ts:41:51 <- test/Lava.spec.js:44:58)
 
    at Generator.next (<anonymous>)
 
    at fulfilled (test/Lava.spec.js:4:58)
 ```

Reply:
Changing delay from 10s to 11s seemed to fix it locally, but I will need to run the suite some more to be somewhat confident about this.

Reply:
Still flaking in this build: https://app.circleci.com/pipelines/github/Coteh/LavaEscape/43/workflows/5f6e9a0d-5555-4ea1-9976-0b71513ec31d/jobs/45/steps

Reply:
Going to merge what's here right now, and continue work on this particular sub-problem in #50